src/prefect/tasks/snowflake/snowflake.py

```diff
@@ -298,6 +299,7 @@ def run(
             "role": role,
             "warehouse": warehouse,
             "autocommit": autocommit,
+            "application": f"Prefect_{self.__class__.__name__}",
```

Comment:
Should we add a comment or include a line in the task's docstring about this? Just in case someone in the future no longer knows what this attribute is for, e.g.
`# required to track task's usage in the Snowflake Partner Network Portal`